freezer/src/freezer.rs

```diff
@@ -72,13 +72,13 @@ impl Freezer {
         &self,
         threshold: BlockNumber,
         get_block_by_number: F,
-    ) -> Result<Vec<FreezeResult>, Error>
+    ) -> Result<BTreeMap<packed::Byte32, (BlockNumber, u32)>, Error>
```

Comment:
This is a public interface. Please add some document about the return type. What's returned in the map on success?